hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -668,13 +669,20 @@ public void createBucket(
       builder.setDefaultReplicationConfig(defaultReplicationConfig);
     }
 
+    ReplicationType replicationType = defaultReplicationConfig == null ?
+        ReplicationType.RATIS : defaultReplicationConfig.getType();
```

Comment:
I don't think the client should make assumptions about default replication on the server.  If the client does not set the bucket's default replication config, it should omit `Replication Type set to ...` from the message.

Reply:
Sure, I will make this change, thanks @adoroszlai

Reply:
Can I just make the similar change like "layoutMsg" ? if the client doesn't set the replication, then we just mention "with server-side default bucket layout" ? @adoroszlai 

 String layoutMsg = bucketLayout != null
        ? "with bucket layout " + bucketLayout
        : "with server-side default bucket layout";

Reply:
@chrisGe123 sounds good, thanks for the suggestion